src/r2mm/manager/PackageDexieStore.ts

```diff
@@ -123,6 +126,40 @@ export async function getPackagesByNames(community: string, packageNames: string
     return packages.map(ThunderstoreMod.parseFromThunderstoreData);
 }
 
+/**
+ * @param community Community which package listings should be used in the lookup.
+ * @param dependencies Lookup targets as Thunderstore dependency strings.
+ * @param useLatestVersion Ignore the version number in dependencyString and return the latest known version.
+ * @returns ThunderstoreCombo[], silently omitting unknown packages and versions.
+ */
+export async function getCombosByDependencyStrings(
+    community: string,
+    dependencyStrings: string[],
+    useLatestVersion=false
+): Promise<ThunderstoreCombo[]> {
+    const split = dependencyStrings.map(splitToNameAndVersion);
+    const keys = split.map((d): [string, string] => [community, d[0]]);
+
+    // Dexie's anyOfIgnoreCase doesn't support compound indexes.
+    const packages = await db.packages.where('[community+full_name]').anyOf(keys).toArray();
+    const versionMap = Object.fromEntries(split);
+
+    return packages.map((rawPackage) => {
+        const rawVersion = useLatestVersion
+            ? rawPackage.versions[0]
```

Comment:
Is version 0 guaranteed to be the latest? Do we sort this on insertion?

I'm aware that we _can_ use the first version and it's unlikely to change, but I don't think it's good practice to rely on this behaviour unless we're accounting for it beforehand. It might be better to either sort, or reduce, to get the latest version instead.

Reply:
The current implementation sure does depend on zero index containing the latest version. So no new sorting or confirmations were added as part of this PR chain. It could be done as a separate task, but IDK how much overhead that would add to Lethal Community. Might be negligible 🤷🏻

Reply:
You can keep it as-is unless you want to do the change. No requirement to sort if we didn't before.

Reply:
Did not change.